api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1184,6 +1184,14 @@ message Cluster {
   // If ``connection_pool_per_downstream_connection`` is true, the cluster will use a separate
   // connection pool for every downstream connection
   bool connection_pool_per_downstream_connection = 51;
+
+  // The idle timeout for connections. The idle timeout is defined as the period in which
+  // there are no active stream. When the idle timeout is reached the connection will be closed.
+  //
+  // .. warning::
+  //   Disabling this timeout has a highly likelihood of yielding connection leaks due to lost TCP
+  //   FIN packets, etc.
```

Comment:
Is there a default if not set? If so how does one disable? Can we document that?

Reply:
great point. the is guard by reloadable flags. If reloadable is on, default is an hour, and setting 0 to disable it. I'll document it. Thanks